tests/sqlx/lexer.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { constructSyntaxTree, parseSqlx } from "@dataform/sqlx/lexer";
+import { expect } from "chai";
+
+describe("@dataform/sqlx", () => {
+  describe("outer SQL lexing", () => {
+    it("backslashes are duplicated so that they act as written in the IDE", () => {
+      // If someone in SQLX SQL writes "\" in the regex, it should be
+      // interpreted as "\".
+      const pairs = [
```

Comment:
`const tests: { in: string; expected: string} = [ { ... }, { ... } ]`

Reply:
👍